src/support_sphere/lib/constants/routes.dart

```diff
@@ -15,14 +15,26 @@ class AppRoute extends Equatable {
 }
 
 class AppNavigation {
-  // TODO: Add body for each route
-  static List<AppRoute> destinations = [
-    const AppRoute(
-        icon: Icon(Ionicons.home_sharp), label: 'Home'),
-    const AppRoute(
-        icon: Icon(Ionicons.person_sharp), label: 'Me', body: ProfileBody()),
-    const AppRoute(
-        icon: Icon(Ionicons.shield_checkmark_sharp), label: 'Prepare'),
-    const AppRoute(icon: Icon(Ionicons.hammer_sharp), label: 'Resources'),
-  ];
+  static List<AppRoute> getDestinations(String? role) {
+    // TODO: Add body for each route
+    List<AppRoute> destinations = [
+      const AppRoute(
+          icon: Icon(Ionicons.home_sharp), label: 'Home'),
+      const AppRoute(
+          icon: Icon(Ionicons.person_sharp), label: 'Me', body: ProfileBody()),
+      const AppRoute(
+          icon: Icon(Ionicons.shield_checkmark_sharp), label: 'Prepare'),
+      const AppRoute(icon: Icon(Ionicons.hammer_sharp), label: 'Resources'),
+    ];
+    if (role == "COM_ADMIN") {
```

Comment:
non-blocking and can definitely be future work, but I wonder if we ought to move these to an enum or constants or something instead of just remembering/using these strings